src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaMetadataService.java

```diff
@@ -0,0 +1,34 @@
+package uk.ac.ebi.atlas.hcalandingpage;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.gson.JsonObject;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.atlas.experiments.ExperimentJsonSerializer;
+
+import java.util.List;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+
+@Component
+public class HcaMetadataService {
+    private final static String ONTOLOGY_IDS = "ontology_ids";
+    private final static String EXPERIMENT_ACCESSIONS = "experiment_accessions";
```

Comment:
These two constants are somewhat unnecessary in my opinion. You’re using them only once, so it adds an unneeded level of indirection (when I read the code I need to go back to these declarations to know what their value is).